lib/registries/addon/components/registries-layout/template.hbs

```diff
@@ -0,0 +1,46 @@
+{{yield (hash
+    heading=(component 'registries-layout/block')
+    sidenavGutterClosed=this.sidenavGutterClosed
+    toggleSidenav=(action this.toggleSidenav)
+    metadataGutterClosed=this.metadataGutterClosed
+    toggleMetadata=(action this.toggleMetadata)
+)}}
+
+<div local-class='ContentBackground'>
+    <Gutters
+        @leftMode={{this.sidenavGutterMode}}
+        @leftClosed={{this.sidenavGutterClosed}}
+        @rightMode={{this.metadataGutterMode}}
+        @rightClosed={{this.metadataGutterClosed}}
+        as |gutters|
+    >
+        {{#unless this.hideLeftGutter}}
```

Comment:
Ideally we'd hide the gutters simply by leaving them out:
```hbs
{{! three column with heading }}
<Layout as |layout|>
  <layout.Heading />
  <layout.LeftNav />
  <layout.Main />
  <layout.Right />
</Layout>

{{! two column without heading }}
<Layout as |layout|>
  <layout.LeftNav />
  <layout.Main />
</Layout>

{{! one column with heading }}
<Layout as |layout|>
  <layout.Heading />
  <layout.Main />
</Layout>
```

For that to work, it might be best to put each in its own sub-component, each of which internally invokes `gutters.foo`, if relevant:
```hbs
{{yield (hash
  Heading=(component 'layout/heading')
}} {{!     ^ or (element 'div'), if that's all we need }}

<Gutters ... as |gutters|>
  {{yield (hash
    LeftNav=(component 'layout/left-nav' gutters=gutters onLinkClicked=(...))
  }}
  {{yield (hash
    Main=(component 'layout/main-column' gutters=gutters)
  }}
  {{yield (hash
    Right=(component 'layout/right-column' gutters=gutters)
  }}
</Gutters>